packages/react/src/components/Authenticator/hooks/useAuthenticator/index.tsx

```diff
@@ -87,24 +93,27 @@ export type InternalAuthenticatorContext = {
  */
 export type Selector = (context: AuthenticatorContext) => Array<any>;
 
-export const useAuthenticator = (selector?: Selector) => {
+const useAuthenticatorService = () => {
   const { service } = React.useContext(AuthenticatorContext);
 
   if (!service) {
-    console.error(
-      'No `Authenticator.Provider` was found above where `useAuthenticator` hook is. ' +
-        'Please ensure you wrap your App with `Authenticator.Provider` like so: ' +
-        'https://ui.docs.amplify.aws/components/authenticator#useauthenticator-hook'
```

Comment:
Removed as url is fragile, and the `Provider`/`Context` pattern in React is well documented, but open to putting it back

Reply:
This is from this PR. https://github.com/aws-amplify/amplify-ui/pull/1657

From this issue https://github.com/aws-amplify/amplify-ui/issues/1651.

It might be worth keeping to help customers from cryptic error messages.

Reply:
I only removed the url since it can easily become out of sync with the docs site or is that your concern?

Reply:
I'm open to either ways. I added a link to docs because it was a major confusion point.

Since `#useauthenticator-hook` part is fragile, I'm wondering if we could still word it like

> Please ensure you wrap your App with `Authenticator.Provider`. See `useAuthenticator` section on https://ui.docs.amplify.aws/components/authenticator

The url without hash will be stable.

Reply:
@wlee221 [Updated](https://github.com/aws-amplify/amplify-ui/pull/1666/commits/474f7e540dd398abbab34beea33db478ba512519)